### PR TITLE
Add easter egg

### PR DESCRIPTION
**Pull request by diogoscf**
Logo changes into a fidget spinner when you click it 7 times.

### CODE REVIEW

File: _layouts/home.html
Comment (Vidminas) [inline _layouts/home.html:175]: This calls the function immediately, right? not used to (function(){})()
also this should only be counted if you click on the school logo itself, not anywhere, because we want the background game to still be playable

Comment (markspolakovs) [reply _layouts/home.html:175]: @HackerManiac https://en.wikipedia.org/wiki/Immediately-invoked_function_expression

File: _layouts/home.html
Comment (markspolakovs) [inline _layouts/home.html:159]: Is it just me, or is countClicks never used? You need to set an `onclick` handler somewhere.

**Review by markspolakovs — CHANGES_REQUESTED** [review]

File: _layouts/home.html
Comment (atfo) [reply _layouts/home.html:159]: Me thinks also.

**Comment by atfo** [comment]
By the way, great use of closures. 😀

File: _layouts/home.html
Comment (diogoscf) [reply _layouts/home.html:159]: I'm an idiot and I forgot to add the onclick in the logo. I'm going to fix that

Comment (diogoscf) [reply _layouts/home.html:159]: Done

File: _layouts/home.html
Comment (markspolakovs) [inline _layouts/home.html:15]: Thank you. Haven't tested, but the code looks okay. Approving, I'll wait for @HackerManiac to approve before merging.

**Review by markspolakovs — APPROVED** [review]

**Comment by markspolakovs** [comment]
Actually, fuck that, merging.

**Pull request merged**
